src/devices/ExplorerHat/Motors.cs

```diff
@@ -75,8 +75,8 @@ internal Motors(GpioController controller, bool shouldDispose = true)
 
             _motorArray = new List<DCMotor.DCMotor>()
             {
-                DCMotor.DCMotor.Create(MOTOR1_SPDPIN, MOTOR1_DIRPIN, _controller),
-                DCMotor.DCMotor.Create(MOTOR2_SPDPIN, MOTOR2_DIRPIN, _controller)
+                DCMotor.DCMotor.Create(MOTOR1_SPDPIN, MOTOR1_DIRPIN, false, _controller),
```

Comment:
this code should not have to change (if you make it default to false and put it as last on the list that shouldn't be needed)